crux-agent/app/core/providers/lmstudio.py

```
@@ -0,0 +1,624 @@
+"""
+LMStudio provider implementation.
+"""
+import json
+from typing import Any, Dict, List, Optional
+
+from tenacity import (
+    retry,
+    retry_if_exception_type,
+    stop_after_attempt,
+    wait_exponential,
```

Ruff (0.12.2)

`tenacity.wait_exponential` imported but unused

Remove unused import: `tenacity.wait_exponential`

(F401)